src/spyglass/spikesorting/v1/curation.py

```diff
@@ -0,0 +1,483 @@
+from typing import List, Union, Dict
+
+import datajoint as dj
+import numpy as np
+import pynwb
+import spikeinterface as si
+import spikeinterface.extractors as se
+import spikeinterface.curation as sc
+
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.common.common_ephys import Raw
+from spyglass.spikesorting.v1.recording import (
+    SpikeSortingRecording,
+)
+from spyglass.spikesorting.v1.sorting import SpikeSorting, SpikeSortingSelection
+
+schema = dj.schema("spikesorting_v1_curation")
+
+valid_labels = ["reject", "noise", "artifact", "mua", "accept"]
+
+
+@schema
+class CurationV1(dj.Manual):
+    definition = """
+    # Curation of a SpikeSorting. Use `insert_curation` to insert rows.
+    -> SpikeSorting
+    curation_id=0: int
+    ---
+    parent_curation_id=-1: int
+    -> AnalysisNwbfile
+    object_id: varchar(72)
+    merges_applied: enum("True", "False")
+    description: varchar(100)
+    """
+
+    @classmethod
+    def insert_curation(
```

Comment:
I added a method called `insert_metric_curation` that does what you describe. I had to import the MetricCuration tables within the method to avoid circular import. Let me know if you think there are better solutions.